packages/react-app/pages/admin/newBatches.jsx

```diff
@@ -0,0 +1,373 @@
+
+  useEffect(() => {
+    async function fetchBuilders() {
+      setIsLoadingBuilders(true);
+      const fetchedBuilders = await axios.get(serverUrl + serverPath);
+
+      const processedBuilders = fetchedBuilders.data
+        .filter(builder => builder.ens !== "austingriffith.eth")
+        .map(builder => ({
+          builder,
+          status: builder.status,
+          stream: builder,
+          builds: builder.builds?.length || 0,
+          socials: builder,
+          userCreated: builderCreated(builder),
+        }));
+
+      setBuilders(processedBuilders);
+      setIsLoadingBuilders(false);
+    }
+
+    fetchBuilders();
+  }, [serverUrl]);
+
+  const BatchFilterComponent = useMemo(() => {
+    const Component = ({ column }) => (
+      <BatchColumnFilter filterValue={column.filterValue} setFilter={column.setFilter} builders={builders} />
+    );
+    Component.displayName = "BatchFilterComponent";
+    return Component;
+  }, [builders]);
+
+  const columns = useMemo(
+    () => [
+      getColumn(
+        "Builder",
+        "builder",
+        ({ value }) => <BuilderAddressCell builder={value} mainnetProvider={mainnetProvider} />,
+        { disableSortBy: true, Filter: EnsColumnFilter, filter: ensFiltering },
+      ),
+      getColumn("Status", "status", ({ value }) => <BuilderStatusCell status={value} />, { disableSortBy: true }),
+      getColumn("Builds", "builds", ({ value }) => <BuilderBuildsCell buildCount={value} />, { sortDescFirst: true }),
+      getColumn(
+        "Stream",
+        "stream",
+        ({ value }) => (value?.graduated?.status ? "" : <StreamTableCell builder={value} />),
+        { disableFilters: true, Filter: BatchFilterComponent, filter: batchFiltering },
+      ),
+      getColumn("Socials", "socials", ({ value }) => <BuilderSocialLinksCell builder={value} isAdmin={isAdmin} />, {
+        disableSortBy: true,
+      }),
+      getColumn("User Created", "userCreated", ({ value }) => <DateWithTooltip timestamp={value} />, {
+        sortAscFirst: true,
+      }),
+    ],
+    [BatchFilterComponent, mainnetProvider, isAdmin],
+  );
+
+  const {
+    getTableProps,
+    getTableBodyProps,
+    headerGroups,
+    prepareRow,
+    page,
+    canPreviousPage,
+    canNextPage,
+    pageOptions,
+    pageCount,
+    gotoPage,
+    nextPage,
+    previousPage,
+    setPageSize,
+    state: { pageIndex, pageSize },
+  } = useTable(
+    {
+      columns,
+      data: builders,
+      initialState: { pageIndex: 0, pageSize: 25, sortBy: useMemo(() => [{ id: "stream", desc: true }], []) },
+    },
+    useFilters,
+    useSortBy,
+    usePagination,
+  );
+
+  useEffect(() => {
+    setAmountBuilders(page.length);
+  }, [page]);
+
+  const ensFilter = headerGroups[0].headers[0];
+  const batchFilter = headerGroups[0].headers[3];
+
+  return (
+    <Container maxW="container.xl">
+      <MetaSeo
+        title="All Builders"
+        description="These are all the builder that are part of the BuidlGuidl"
```

Comment:
we can probably remove this, since this is a admin page

Reply:
Makes sense